config-example.yaml

```diff
@@ -26,5 +28,24 @@ videoSettings:
   # Change this value and keep it secure.  Treat it like a password to your live stream.
   streamingKey: abc123
 
+  streamQualities:
+    - low:
+      videoBitrate: 800
+      scaledWidth: 640
+      encoderPreset: veryfast
+
+    - medium:
+      videoBitrate: 1200
+      encoderPreset: veryfast
+
+    - high:
+      videoBitrate: 1600
+      encoderPreset: veryfast
+
 # Set to true if you don't want the service checking for future releases.
 disableUpgradeChecks: false
+
+# Enable if you want your stream to be listed in the Owncast directory.
```

Comment:
Since we're not ready to publicly release the directory, I've left these out of the config examples.  It's probably best not to encourage people to actively register with the directory yet since it hasn't been tested much.  But if people dig through the config struct and figure it out themselves, then they can go for it :)

Reply:
Fair enough, updating the PR.